bootstrap/bootstrap.sh

```diff
@@ -45,6 +45,7 @@ for cluster in ${6}; do
   ./scripts/install-sealed-secrets.sh "$@"|| error_exit "ERROR: Unable to install sealed secrets"
   ./scripts/create-flux-githubkey-secret.sh "$@"|| error_exit "ERROR: Unable to create flux githubkey secret"
   ./scripts/install-flux.sh "$@"|| error_exit "ERROR: Unable to install flux"
+  [[ $3 =~ ^(sbox|preview|ithc|perftest|aat|demo)$ ]] && (./scripts/disable-container-insights.sh "$@" || error_exit "ERROR: Unable to disable container application insights")
```

Comment:
missing some environments 
- sbox-ptl 
can you check any additional

Reply:
Added sbox-ptl